src/snap-confine.apparmor.in

```diff
@@ -107,7 +107,7 @@
 
     # mount calls to setup the pivot_root based chroot with the core snap as
     # the root filesystem.
-    mount options=(rw bind) @SNAP_MOUNT_DIR@/ubuntu-core/*/ -> /tmp/snap.rootfs_*/,
+    mount options=(rw bind) @SNAP_MOUNT_DIR@/{ubuntu-,}core/*/ -> /tmp/snap.rootfs_*/,
```

Comment:
For consistency with other rules with alternation, can we use the equivalent `{,ubuntu-}` here and elsewhere instead?